lib/oban/worker.ex

```diff
@@ -254,6 +269,78 @@ defmodule Oban.Worker do
           if MyApp.something?(job), do: :ok, else: {:snooze, 60}
         end
       end
+
+  ## Workers in A Different Application
+
+  Occasionally you may need to insert a job for a worker that exists in another
+  application. In that case you can use `Oban.Job.new/2` to build the changeset
+  manually:
+
+      %{id: 1, user_id: 2}
+      |> Oban.Job.new(queue: :default, worker: OtherApp.Worker)
+      |> Oban.insert()
+
+  ## Scheduling Jobs
+
+  You can schedule jobs down to the second any time in the future:
+
+      %{id: 1}
+      |> MyApp.Business.new(schedule_in: _seconds = 5)
+      |> Oban.insert()
+
+  Jobs may also be scheduled at a specific timestamp in the future:
+
+      %{id: 1}
+      |> MyApp.Business.new(scheduled_at: ~U[2020-12-25 19:00:56.0Z])
+      |> Oban.insert()
+
+  Scheduling is _always_ in UTC. You'll have to shift timestamps in other zones to
+  UTC before scheduling:
+
+      %{id: 1}
+      |> MyApp.Business.new(scheduled_at: DateTime.shift_zone!(datetime, "Etc/UTC"))
+      |> Oban.insert()
+
+  ### Caveats & Guidelines
+
+  Usually, scheduled job management operates in `global` mode and notifies queues
+  of available jobs via PubSub to minimize database load. However, when PubSub
+  isn't available, staging switches to a `local` mode where each queue polls
+  independently.
+
+  Local mode is less efficient and will only happen if you're running in an
+  environment where neither `Postgres` nor `PG` notifications work. That situation
+  should be rare and limited to the following conditions:
+
+    1. Running with a connection pooler, like [`pg_bouncer`][pg_bouncer],
+       in transaction mode.
+    2. Running without clustering, that is, without *distributed Erlang*.
+
+  If **both** of those criteria apply and PubSub notifications won't work, then
+  staging will switch to polling in `local` mode.
```

Comment:
Do you think there a better location for this section? Maybe a guide about distribution/pubsub/notifiers? You're in the head-space for it more than we are right now.

Reply:
Yes, we can split this out. The nice thing about splitting it out is that we'll have a home for this stuff (and a URL for it too).